TKeazirian.HTTPServer/Controller.cs

```diff
@@ -4,15 +4,24 @@ public class Controller
     {
         private const string NewLine = "\r\n";
 
-        public string EchoRequestBody(string request)
+        public string EchoRequestBody(string requestBody)
```

Comment:
The controller is now controlling 3 different endpoints. Is it time to extract a few classes?